Data-Transfer-ReactJS-Dashboard/src/pages/Layout.js

```diff
@@ -0,0 +1,31 @@
+import { Outlet, Link} from "react-router-dom";
+import './Layout.css';
+
+const Layout = () => {
+  return (
+    <>
+      <br></br>
+      <br></br>
```

Comment:
How about using css for spacing rather than multiple line breaks

Reply:
used CSS instead of multiple breaks.